data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -85,6 +85,216 @@
       "actionid": 13,
       "actioncode": "",
       "tenantId": "statea"
-    }
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 21,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 22,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 36,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 37,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 38,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUSTER_ROLL_VERIFIER",
+      "actionid": 37,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUSTER_ROLL_VERIFIER",
+      "actionid": 38,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 39,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 40,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 42,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 44,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 45,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 46,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 50,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
+      "actionid": 71,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUSTER_ROLL_VERIFIER",
```

Comment:
MUSTER_ROLL_VERIFIER require 37 action

Reply:
Line no 120 to 123 ,Having MUSTER_ROLL_VERIFIER for 37 action